keyboards/silverbullet44/keymaps/FKML/keymap.c

```diff
@@ -0,0 +1,169 @@
+/* Copyright 2019 SwanMatch
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include QMK_KEYBOARD_H
+
+// Defines the keycodes used by our macros in process_record_user
+//enum custom_keycodes {
+//  QMKBEST = SAFE_RANGE,
+//  QMKURL
+//};
+
+#define _QWERTY 0
+#define _LOWER 1
+#define _RAISE 2
+#define _ADJUST 3
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  LOWER,
+  RAISE,
+  ADJUST,
+};
+#define _______ KC_TRNS
+#define XXXXXXX KC_NO
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+/* QWERTY
+ * ,-----------------------------------------.                                  ,-----------------------------------------.
+ * | Tab  |   Q  |   W  |   E  |   R  |   T  |                                  |   Y  |   U  |   I  |   O  |   P  |  =   |
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * |Shift |   A  |   S  |   D  |   F  |   G  |                                  |   H  |   J  |   K  |   L  |   ;  |  '   |
+ * |------+------+------+------+------+------|                                  |------+------+------+------+------+------|
+ * | CMD  |   Z  |   X  |   C  |   V  |   B  |-------.-------.  ,---------------|   N  |   M  |   ,  |   .  |   /  |  -   |
+ * `-----------------------------------------/       /       /   \       \      \-----------------------------------------'
+ *                            | CAPS  | LOWER| Space/  Bcsp /     \    .  \ Ent | RAISE|LEDOff|
+ *                            `--------------- -------------'       '--------------------------'
+ */
+ [_QWERTY] = LAYOUT( \
```

Comment:
なんか全体的に行末のバックスラッシュいらないって言われた。。。
(魔王知らなかった)

```suggestion
 [_QWERTY] = LAYOUT(
```